src/controllers/auth-login-controller.js

```diff
@@ -0,0 +1,8 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('angular-token-auth.auth-login', []);
```

Comment:
I know we should have picked this up before, but I think these modules should be named more descriptively and consistently with the file names. e.g. `angular-token-auth.auth-login-controller`